### ISSUE TITLE
Request: Synchronize "current" watched status

### ISSUE THREAD

**Issue by Fmstrat** [jacobwgillespie/plex-sync#44]
Right now, plex-sync defaults to "watched" if the view count exists and/or is greater than 0:

https://github.com/jacobwgillespie/plex-sync/blob/master/src/plex.js#L85

However, I think it would be better to actually query the current watched status. For instance, if I mark something unwatched, have that status sync over. I'm not sure this is even possible the way Plex updates the XML file, but figured I would bring it up in case you know something I don't.

**Comment by astrodad** [comment]
I just set bunch of episodes to watched manually and it synced just fine.

**Comment by WillGunn** [comment]
@astrodad I believe he is saying that if he marks an item unwatched on a server, because that item was the most recent one updated, use that as the point of truth in the sync.  I just don't know if plex keeps track of when something was marked watched vs unwatched.  Is that right @Fmstrat ?

**Comment by Fmstrat** [comment]
@WillGunn correct. It's when we set something to "unwatched" that it doesn't work. The "watched" status always takes precedence, vs the time. The only way to do this would be to give plex-sync it's own database, though, as Plex doesn't store the time modified. It would be interesting to integrate this into something like PlexPy, which already does the tracking required.
